website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 90
+---
+## General
+
+### How does a provisioner decide to manage a particular node?
+Karpenter only provisions nodes for pods with a status condition `Unschedulable=True`
+Karpenter will only take action on nodes that it provisions.
+All nodes launched by Karpenter will be labeled with `karpenter.sh/provisioner-name`.
+
+### What cloud providers are supported?
+AWS is the first cloud provider supported by Karpenter, although it is designed to be used with other cloud providers as well.
+See [[Cloud provider]({{< ref "/docs/concepts/#cloud-provider" >}}) for details.
+
+### What deployment methods are supported by Karpenter?
+To deploy Karpenter manually, see [[Getting Started]({{< ref "/docs/getting-started/" >}}).
+To deploy using Terraform, see [[Getting Started with Terraform]({{< ref "/docs/getting-started-with-terraform/" >}}).
+To deploy using kOps, see [[Getting Started with kOps]({{< ref "/docs/getting-started-with-kops/" >}}).
+
+### Can I write my own cloud provider for Karpenter?
+!!! NEEDS INFO !!!
```

Comment:
Yes, but there's no docs for it. Devs can start with https://github.com/aws/karpenter/tree/main/pkg/cloudprovider to see how the AWS provider is built but there are other sections of the code that will require changes too.